docs/8-custom-actions.md

```diff
@@ -30,6 +30,44 @@ end
 This collection action will generate a route at `/admin/posts/import_csv`
 pointing to the `Admin::PostsController#import_csv` controller action.
 
+To paginate a table using ajax within a page, the following aproach works with Kaminari
+
+```ruby
+ActiveAdmin.register Post do
+  collection_action :member_posts, method: :get do
+    render 'admin/members/member_posts', context: self
+  end
+
+  span id: 'member_posts' do
+    render partial: 'admin/members/member_posts'
+  end
```

Comment:
I wanted to try this, but I'm not sure in which context this is supposed to be rendered. Is it in the show page? :thinking:

Reply:
the above partial (for which an example is below) is an #index - render an index of all `posts` for a `member`

Reply:
But I then you need to wrap the `span` helper with an `index do ... end` block, right?